### PR TITLE
fix: TypeError: Cannot read property 'enabled' of undefined (fixes #369)

### LINKED ISSUE #369: [blendid] quick start fails
env:

* RHEL 7.3
* `node --version`:  `v7.3.0`
* `yarn --version`: `0.18.1`
* `cat package.json | grep blendid`: `    "blendid": "^4.0.0-blendid-beta-15"`

I followed the quick start steps:

	yarn init
	yarn add blendid
	yarn run blendid -- init
	yarn run blendid

the last command fails with:

	yarn run blendid
	yarn run v0.18.1
	$ "/home/pb/blend/node_modules/.bin/blendid" 
	[22:21:43] Working directory changed to ~/blend/node_modules/blendid
	[22:21:44] Using gulpfile ~/blend/node_modules/blendid/gulpfile.js
	[22:21:44] Starting 'default'...
	[22:21:44] Starting 'clean'...
	[22:21:44] Finished 'clean' after 7.88 ms
	[22:21:44] Starting 'fonts'...
	[22:21:44] Starting 'images'...
	[22:21:44] Starting 'svgSprite'...
	[22:21:44] Finished 'svgSprite' after 16 ms
	[22:21:44] Finished 'fonts' after 34 ms
	[22:21:44] Finished 'images' after 23 ms
	[22:21:44] Starting 'html'...
	[22:21:44] Starting 'stylesheets'...
	[22:21:44] Finished 'html' after 46 ms
	[22:21:45] Finished 'stylesheets' after 112 ms
	[22:21:45] Starting 'browserSync'...
	[22:21:45] 'browserSync' errored after 1.56 ms
	[22:21:45] TypeError: Cannot read property 'enabled' of undefined
	    at module.exports (/home/pb/blend/node_modules/blendid/gulpfile.js/lib/webpack-multi-config.js:62:35)
	    at Gulp.browserSyncTask (/home/pb/blend/node_modules/blendid/gulpfile.js/tasks/browserSync.js:12:23)
	    at module.exports (/home/pb/blend/node_modules/orchestrator/lib/runTask.js:34:7)
	    at Gulp.Orchestrator._runTask (/home/pb/blend/node_modules/orchestrator/index.js:273:3)
	    at Gulp.Orchestrator._runStep (/home/pb/blend/node_modules/orchestrator/index.js:214:10)
	    at Gulp.Orchestrator.start (/home/pb/blend/node_modules/orchestrator/index.js:134:8)
	    at /home/pb/blend/node_modules/gulp-sequence/index.js:68:12
	    at apply (/home/pb/blend/node_modules/thunks/thunks.js:362:38)
	    at tryRun (/home/pb/blend/node_modules/thunks/thunks.js:232:19)
	    at runThunk (/home/pb/blend/node_modules/thunks/thunks.js:225:15)
	error Command failed with exit code 1.
	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### PR DESCRIPTION
**Pull request by typekpb**

### CODE REVIEW

File: gulpfile.js/lib/webpack-multi-config.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ module.exports = function(env) {
     webpackConfig.output.pathinfo = true
 
     // Create new entries object with webpack-hot-middleware and react-hot-loader (if enabled)
-    if(TASK_CONFIG.javascripts.hot.enabled !== false) {
+    if(TASK_CONFIG.javascripts.hot && TASK_CONFIG.javascripts.hot.enabled !== false) {
```
Comment (greypants) [inline gulpfile.js/lib/webpack-multi-config.js]: Oops. I think this should actually be: 

```js
if(!TASK_CONFIG.javascripts.hot || TASK_CONFIG.javascripts.hot.enabled !== false`)
```

Basically I want hot-loading enabled by default, unless you specifically turn it off. So if there's no config for it, assume it's on.

**Comment by greypants** [comment]
Thanks for catching and reporting that!

File: gulpfile.js/lib/webpack-multi-config.js (hunk 1 above)
Comment (typekpb) [inline gulpfile.js/lib/webpack-multi-config.js]: OK, fixed. Moreover addressed a couple more there. Let's see if I didn't mess it up even more :)

**Comment by greypants** [comment]
published `4.0.0-blendid-beta-16`

**Comment by typekpb** [comment]
Perfect, thanks!

**Pull request merged**